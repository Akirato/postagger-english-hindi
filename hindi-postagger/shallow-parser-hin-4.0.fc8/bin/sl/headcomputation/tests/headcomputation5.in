<Sentence id="1">
1	((	VGF	
1.1	xAzdZiyZe	VM	<fs af='xAzdZA,v,,,any,,iyZe,iyZe'>
	))		
2	((	NP	
2.1	Weke	PSP	<fs af='Weke,psp,,,,,,'>
	))		
3	((	JJP	
3.1	abAka	JJ	<fs af='abAka,adj,,,,,,'>
	))		
4	((	VGF	
4.1	hayZe	VM	<fs af='ha,v,,,any,,e,e'>
	))		
5	((	VGF	
5.1	xeKawe	VM	<fs af='xeK,v,,,7,,A,A'>
5.2	WAki	VAUX	<fs af='WAk,v,,,1,,e,e'>
	))		
6	((	NP	
6.1	xi	XC	<fs af='xi,unk,,,,,,' poslcat="NM">
	))		
7	((	NP	
7.1	greta	NN	<fs af='greta,unk,,,,,,' poslcat="NM">
	))		
8	((	NP	
8.1	himAlayZAna	NN	<fs af='himAlayZAna,adj,,,,,,' poslcat="NM">
8.2	reFja	NNP	<fs af='reFja,n,,sg,,d,0,0'>
	))		
9	((	VGF	
9.1	.	SYM	<fs af='.,punc,,,,,,' poslcat="NM">
	))		
</Sentence>
