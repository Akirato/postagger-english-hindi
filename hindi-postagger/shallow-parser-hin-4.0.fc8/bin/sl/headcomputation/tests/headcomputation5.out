<Sentence id="1">
1	((	VGF	<fs af='xAzdZA,v,,,any,,iyZe,iyZe' head=xAzdZiyZe>
1.1	xAzdZiyZe	VM	<fs af='xAzdZA,v,,,any,,iyZe,iyZe' name=xAzdZiyZe>
	))		
2	((	NP	<fs af='Weke,psp,,,,,,' head=Weke>
2.1	Weke	PSP	<fs af='Weke,psp,,,,,,' name=Weke>
	))		
3	((	JJP	<fs af='abAka,adj,,,,,,' head=abAka>
3.1	abAka	JJ	<fs af='abAka,adj,,,,,,' name=abAka>
	))		
4	((	VGF	<fs af='ha,v,,,any,,e,e' head=hayZe>
4.1	hayZe	VM	<fs af='ha,v,,,any,,e,e' name=hayZe>
	))		
5	((	VGF	<fs af='xeK,v,,,7,,A,A' head=xeKawe>
5.1	xeKawe	VM	<fs af='xeK,v,,,7,,A,A' name=xeKawe>
5.2	WAki	VAUX	<fs af='WAk,v,,,1,,e,e' name=WAki>
	))		
6	((	NP	<fs af='xi,unk,,,,,,' poslcat="NM" head=xi>
6.1	xi	XC	<fs af='xi,unk,,,,,,' name=xi poslcat="NM">
	))		
7	((	NP	<fs af='greta,unk,,,,,,' poslcat="NM" head=greta>
7.1	greta	NN	<fs af='greta,unk,,,,,,' name=greta poslcat="NM">
	))		
8	((	NP	<fs af='reFja,n,,sg,,d,0,0' head=reFja>
8.1	himAlayZAna	NN	<fs af='himAlayZAna,adj,,,,,,' name=himAlayZAna poslcat="NM">
8.2	reFja	NNP	<fs af='reFja,n,,sg,,d,0,0' name=reFja>
	))		
9	((	VGF	<fs af='.,punc,,,,,,' poslcat="NM" head=.>
9.1	.	SYM	<fs af='.,punc,,,,,,' name=. poslcat="NM">
	))		
</Sentence>
